spanner/integration_test.go

```diff
@@ -2943,6 +2943,130 @@ func TestIntegration_DML(t *testing.T) {
 	}
 }
 
+func TestIntegration_DMLReturning(t *testing.T) {
```

Comment:
@c2nes: Can we add end-to-end tests for running DML statement with returning clause using `tx.Update` and `tx.BatchUpdate` as well.

Reply:
Done.